docs/java-tutorial-2-graphx.md

```diff
@@ -0,0 +1,170 @@
+# Run existing jars developed for Apache GraphX on GraphScope
+
+[Apache Spark](https://spark.apache.org/) is a famous engine for large-scale data analytics. [Spark Graphx](https://spark.apache.org/graphx/) is Spark's graph
+computing module, which provides  flexible and efficient graph computation framework.
+
+Graphscope is also developed to integrated with Spark Graphx. User can easily deploy a graphscope cluster colocated with spark cluster. And by switch `SparkSession` to `GSSparkSession`, user can experience up to 7 times performance 
+improvment when running graphx algorithms.
+
+## Deploy GraphScope along with Spark
+
+We assume you already have a spark cluster deployed. If you don't have a spark cluster deployed, please refer to [spark-cluster-overview](https://spark.apache.org/docs/latest/cluster-overview.html) to deploy a spark cluster.
+
+Also, GraphScope can be easily distributed with python package. Since GraphScope only 
+support python3, you shall upgrade your python enviroment before proceeding on.
+
+Then, on client side, execute command below to create a python enviroment pack which contains graphscope package.
+
+```bash
+python -m venv pyspark_venv
+source pyspark_venv/bin/activate
+pip3 install graphscope
+venv-pack -o pyspark_venv.tar.gz
+```
+
+Now, `pyspark_venv.tar.gz` contains neccessary enviroments graphscope need. Every time
+you submit a job to your spark cluster, remember to upload this pack.
+
+```bash
+export PYSPARK_DRIVER_PYTHON=python # Do not set in cluster modes.
+export PYSPARK_PYTHON=./environment/bin/python
+spark-submit --archives pyspark_venv.tar.gz#environment ...
+```
+
+## Run example GraphX apps
+
+Several GraphX algorithms are also contained in [grape-demo.jar](https://graphscope.oss-cn-beijing.aliyuncs.com/jar/grape-demo-0.18.0-shaded.jar).
```

Comment:
I just wonder is this the Java way? maybe a central managed jar, import to their pom, would be better?

Reply:
I am convinced.